src/Tickets/Commerce/Provider.php

```diff
@@ -61,6 +61,9 @@ public function register() {
 		$this->container->singleton( Notice_Handler::class );
 
 		$this->container->singleton( Module::class );
+		// We need to init for the registration as a module to take place early.
+		$this->container->get( Module::class );
```

Comment:
```suggestion
		// We need to init for the registration as a module to take place early.
		$this->container->singleton( Module::class, Module::class );
```

Reply:
Why ? if the the 2 first arguments are the same this is completely unnecessary. The singleton method already does that.

Reply:
Passing as 2nd param loads it immediately.

Reply:
Nope, i dont see it :(